project-base/tests/FrontendApiBundle/Functional/Product/ProductsFilteringOptionsTest.php

```diff
@@ -0,0 +1,251 @@
+                                count
+                                isAbsolute
+                            },
+                            inStock,
+                            minimalPrice,
+                            maximalPrice,
+                            parameters {
+                                name
+                                values {
+                                    text
+                                    count
+                                    isAbsolute
+                                }
+                            }
+                        }
+                    },
+                }
+            }
+        ';
+
+        $minimalPrice = $this->getFormattedMoneyAmountConvertedToDomainDefaultCurrency('318.75');
+        $maximalPrice = $this->getFormattedMoneyAmountConvertedToDomainDefaultCurrency('21590');
+
+        $expectedResult = '{
+    "data": {
+        "category": {
+            "products": {
+                "productFilterOptions": {
+                    "flags": [
+                        {
+                            "flag": {
+                                "name": "' . t('Action', [], 'dataFixtures', $this->firstDomainLocale) . '"
+                            },
+                            "count": 1,
+                            "isAbsolute": true
+                        },
+                        {
+                            "flag": {
+                                "name": "' . t('TOP', [], 'dataFixtures', $this->firstDomainLocale) . '"
+                            },
+                            "count": 2,
+                            "isAbsolute": true
+                        }
+                    ],
+                    "brands": [
+                        {
+                            "brand": {
+                                "name": "' . t('A4tech', [], 'dataFixtures', $this->firstDomainLocale) . '"
+                            },
+                            "count": 1,
+                            "isAbsolute": true
+                        },
+                        {
+                            "brand": {
+                                "name": "' . t('LG', [], 'dataFixtures', $this->firstDomainLocale) . '"
+                            },
+                            "count": 1,
+                            "isAbsolute": true
+                        },
+                        {
+                            "brand": {
+                                "name": "' . t('Philips', [], 'dataFixtures', $this->firstDomainLocale) . '"
+                            },
+                            "count": 1,
+                            "isAbsolute": true
+                        },
+                        {
+                            "brand": {
+                                "name": "' . t('Sencor', [], 'dataFixtures', $this->firstDomainLocale) . '"
+                            },
+                            "count": 1,
+                            "isAbsolute": true
```

Comment:
Can you please add a test for filteringOptions while some filters are applied? 
To ensure `isAbsolute` and counts works fine when I filter the products.

Reply:
60126d9bb888d2501ea800e22598614dbefb848d